### PR TITLE
Update Chair, SquareTable, and RectangleTable2 to add chair positioning and create CircleTable2 with chairs around border

### PR DESCRIPTION
**Pull request by rogers-ch**
Chair component updates:

Round Chair style for CircleTable2:
image

Example of chair styles for RectangleTable2 and SquareTable:
image
image

CircleTable2 component:
image

RectangleTable2 component:
image

SquareTable component updates:
image

### CODE REVIEW

**Comment by ralph-dev** [comment]
@rogers-ch can you attach the screenshots in the Pr

File: src/Containers/Chair.tsx
Comment (ralph-dev) [inline src/Containers/Chair.tsx:10]: No need to say Boolean value, it's implied by typescript type this is repetition

File: src/Containers/Chair.tsx
Comment (ralph-dev) [inline src/Containers/Chair.tsx:26]: Remove Boolean value

File: src/Containers/Chair.tsx
Comment (ralph-dev) [inline src/Containers/Chair.tsx:43]: Put this inside the return statement it looks out of place, 
Do
IsRound ?
GetCurvedCircleChair
:
GetStraightChair

**Review by ralph-dev — COMMENTED** [review]
First round of comments ^

File: src/Containers/Chair.tsx
Comment (rogers-ch) [reply src/Containers/Chair.tsx:91]: We're pulling colors from useTheme(). When we tried accessing useTheme() in the styled component, the components wouldn't render in Storybook anymore. Storybook sticks in the loading phase. Is there a particular way to access useTheme() in the styled components?

Comment (ralph-dev) [reply src/Containers/Chair.tsx:91]: You cannot useTheme in the styled component

Comment (ralph-dev) [reply src/Containers/Chair.tsx:91]: Your supposed to do
${(props) => props.theme}

Comment (ralph-dev) [reply src/Containers/Chair.tsx:91]: OR
${({theme}) => theme.Colors etc}

File: src/Containers/CircleTable2.tsx
Comment (ralph-dev) [inline src/Containers/CircleTable2.tsx:216]: This 3 is a magic number

File: src/Containers/CircleTable2.tsx
Comment (ralph-dev) [inline src/Containers/CircleTable2.tsx:219]: There is quite a bit of logic why not put it all into a function?
Also if you are doing this much logic, chances are your doing something wrong

**Review by ralph-dev — APPROVED** [review]
LGTM, note how the repo no longer uses the Stories Folder as the Component Folder maintains both.

**Comment by ralph-dev** [comment]
Follow this guide to fix the merge issues btw
https://wolfbeacon.atlassian.net/wiki/spaces/CW/pages/1683030053/3.1+Merge+Conflicts

File: src/Containers/Chair.tsx
Comment (ralph-dev) [inline src/Containers/Chair.tsx:124]: Group all of these under relative size, way too much variable destructing

File: src/Containers/Chair.tsx
Comment (ralph-dev) [inline src/Containers/Chair.tsx:134]: Same issue , group together

File: src/Containers/CircleTable.tsx
Comment (ralph-dev) [inline src/Containers/CircleTable.tsx:71]: Should this live here or in the styled component? If this only affects styling it should be in styled components

File: src/Containers/CircleTable.tsx
Comment (ralph-dev) [inline src/Containers/CircleTable.tsx:105]: Key for what
generateChairKeys??

File: src/Containers/CircleTable.tsx
Comment (ralph-dev) [inline src/Containers/CircleTable.tsx:102]: prefix
not pre?

File: src/Containers/CircleTable.tsx
Comment (ralph-dev) [inline src/Containers/CircleTable.tsx:150]: MIN_CHAIRS_BEFORE_TABLE_RESIZE
MIN_CHAIRS_TABLE_RESIZE_BREAKPOINT 
use SCREMAING_SNAKE_CASE for consts always when static const (Never Changing)

File: src/Containers/SquareTable/SquareTable.tsx
Comment (ralph-dev) [inline src/Containers/SquareTable/SquareTable.tsx:195]: Living in the wrong place, Styled component Logic

File: src/Containers/SquareTable/SquareTable.tsx
Comment (ralph-dev) [inline src/Containers/SquareTable/SquareTable.tsx:282]: Group

**Review by ralph-dev — CHANGES_REQUESTED** [review]
Lots of Styling needs to be grouped together so that
1) You can write LESS code
2) Code is more maintainable

File: src/Containers/CircleTable.tsx
Comment (ralph-dev) [inline src/Containers/CircleTable.tsx:176]: why is there random enter lines

File: src/Containers/SquareTable/SquareTable.tsx
Comment (ralph-dev) [inline src/Containers/SquareTable/SquareTable.tsx:249]: random enter lines?

File: src/Containers/SquareTable/_ChairRow.tsx
Comment (ralph-dev) [inline src/Containers/SquareTable/_ChairRow.tsx:163]: This -15 is a magic number what does it mean?
WHITESPACE_BETWEEN_CHAIR_AND_TABLE?

**Review by ralph-dev — CHANGES_REQUESTED** [review]
Still Magic Numbers in the Code, not all of them just specific ones

**Comment by ralph-dev** [comment]
Also there is merge conflicts ^, Merge Master into this Branch fix conflicts, then wait for approval so you can merge into master

**Review by ralph-dev — APPROVED** [review]
LGTM we will do the fixes in another PR

**Review by ralph-dev — APPROVED** [review]
LGTM, you can merge

**Pull request merged**
